hydromt_wflow/workflows/demand.py

```diff
@@ -602,3 +691,78 @@ def irrigation(
         ds_irrigation["paddy_irrigation_trigger"] = trigger.where(paddy_areas == 1, 0)
 
     return ds_irrigation
+
+
+def irrigation_from_vector(
+    gdf_irrigation: gpd.GeoDataFrame,
+    ds_like: xr.Dataset,
+    cropland_class: List[int],
+    paddy_class: List[int] = [],
+    area_threshold: float = 0.6,
+    lai_threshold: float = 0.2,
+    logger=logger,
+):
+    """
+    Prepare irrigation maps for paddy and non paddy from geodataframe.
+
+    Parameters
+    ----------
+    da_irrigation: gpd.GeoDataFrame
+        Irrigation map
```

Comment:
```suggestion
    gdf_irrigation: gpd.GeoDataFrame
        Irrigation map
```

Instead of irrigation map, it should be irrigation shapefile or something, right?

Reply:
Indeed! Forgot to replace when I copy/pasted :) I'll change it.